CMakeLists.txt

```diff
@@ -88,7 +88,7 @@ endif()
 
 include(Flake8)
 
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 23)
```

Comment:
I forgot to change this in the previous PR.

Reply:
Huh, I thought it was intentional. :)